redfish-core/lib/pcie.hpp

```diff
@@ -482,6 +598,40 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                     return;
                 }
             });
+
+    BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/PCIeDevices/<str>/")
+        .privileges(redfish::privileges::patchPCIeDevice)
+        .methods(boost::beast::http::verb::patch)(
+            [](const crow::Request& req,
+               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+               const std::string& device) {
+                std::optional<nlohmann::json> oem;
+                if (!json_util::readJson(req, asyncResp->res, "Oem", oem))
+                {
+                    return;
```

Comment:
Need a 

                BMCWEB_LOG_ERROR << "Property Value Format Error ";
                messages::propertyValueFormatError(
                                asyncResp->res, *openbmc,
                                "OpenBMC");
                return;
                
here?

Reply:
Actually readJson handles this pretty well